### DISCUSSION TITLE
Add a Web version

### DISCUSSION THREAD

**Discussion by Samdan9** [nzp-team/nzportable#365] [General]
Do it you won't no balls

(this is a joke i know this will be a pain in the ass since webquake probably doesnt support any of the tools used to make nz:p work the way it does)

**Comment by MotoLegacy** [comment]
We have one. https://nzp.gay/

**Reply by Samdan9** [reply]
Well- there went my epic troll plan (i love this project btw i play it on my 3DS ALL THE TIME! honestly great work!

**Reply by Samdan9** [reply]
ps: i probably should had checked under releases i just assumed that there wasnt smh

**Reply by MotoLegacy** [reply]
Thanks for the support. Sorry about the state of the 3DS build. It's getting my full attention next month, this month is all about VITA/Switch.

(insert patreon shill)
